features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -1377,6 +1377,17 @@
   "console.server.scopes.read": ["internal_governance_view"],
   "console.server.scopes.update": ["internal_governance_update"],
   "console.server.scopes.delete": [],
+  "console.tenants.enabled": true,
+  "console.tenants.disabled_features": [
+    "tenants.add.tenant.from.dropdown",
+    "tenants.make.default",
+    "tenants.organizations.quick.nav.from.dropdown"
+  ],
+  "console.tenants.scopes.feature": ["console:tenants"],
+  "console.tenants.scopes.create": ["internal_notification_senders_create"],
```

Comment:
Don't we need to have unique scopes for tenant mgt CRUD?

Reply:
Made a mistake. Fixed now.